elifepubmed/generate.py

```diff
@@ -364,6 +365,27 @@ def set_abstract(parent, poa_article, abstract_label_types):
         set_abstract_text(abstract_tag, '', '')
 
 
+def set_plain_language_summary(parent, article):
+    "set an OtherAbstract tag to include the digest as plain-language-summary"
+    tag_name = 'OtherAbstract'
+    if hasattr(article, 'digest') and article.digest:
+        # Pubmed allows <i> tags, not <italic> tags
+        tag_converted_digest = eautils.replace_tags(article.digest, 'italic', 'i')
```

Comment:
perhaps have a dictionary to loop through?

Reply:
I added an `OrderedDict()` of these tag replacements into `utils.py`, and a new function to replace each of the tags.